src/main/java/com/powsybl/ws/commons/computation/service/AbstractWorkerService.java

```diff
@@ -143,15 +141,12 @@ public Consumer<Message<String>> consumeRun() {
                     sendResultMessage(resultContext, result);
                     LOGGER.info("{} complete (resultUuid='{}')", getComputationType(), resultContext.getResultUuid());
                 }
-            } catch (InterruptedException e) {
-                Thread.currentThread().interrupt();
+            } catch (CancellationException e) {
+                LOGGER.info("Computation was interrupted");
```

Comment:
Better to add context (ids or something) to this log just like above it ?

Reply:
this is probably a redundant log? we already log the cancellation elsewhere:
```
c.p.w.c.c.service.AbstractWorkerService  : Computation was interrupted
....
....
....
c.p.w.c.c.service.AbstractWorkerService  : Security analysis was canceled (resultUuid='49409bd8-59d5-45ef-adcc-ff8293f4dc25')
c.p.o.sa.AbstractSecurityAnalysis        : Security analysis cancelled in 8321 ms
```